### DISCUSSION TITLE
Allow controling GPIO via GCODE

### DISCUSSION THREAD

**Discussion by Fapu** [catgiggle/OctoPrint-GpioControl#11] [Ideas]
Create an input for a GCODE Command. If the Command using in Commandline oder Printing Programm, toggle or set the state of a GPIO PIN.

Is this possible?

**Comment by catgiggle** [comment]
Please take a note at GCODE System Commands plugin (https://plugins.octoprint.org/plugins/gcodesystemcommands/). It does exactly what you need.
